pose-detection/src/blazepose/detector_utils.ts

```diff
@@ -53,12 +53,10 @@ export function validateModelConfig(modelConfig: BlazeposeModelConfig):
 
 export function validateEstimationConfig(
     estimationConfig: BlazeposeEstimationConfig): BlazeposeEstimationConfig {
-  let config;
+  let config = estimationConfig;
```

Comment:
Same thing. I don't know how JS handles it, but doesn't look like copy of an object.